### ISSUE TITLE
launch-time issue reported by Tobi developers

### ISSUE THREAD

**Issue by rdeltour** [daisy/pipeline#513]
A while ago Tobi developers reported (via email) some instability when launching the Pipeline 2 web service on Windows (from Tobi, but also from the command line).

Several core framework libraries have been updated  in 1.10.2, and @bertfrees fixed some instability issues (that were detected when running the Braille modules). See daisy/pipeline-tasks#84

@marisademeglio any progress on Tobi's issue? We need to check the status of this once 1.10.2 is released.

**Comment by marisademeglio** [comment]
I will check with them again but the last time we discussed it (March 2017), I was waiting for clarification and a log file from them.

**Comment by marisademeglio** [comment]
The report from them is: "Okay, so I installed Pipeline and Tobi from the official distributions, and I successfully ran a basic conversion test.
I'm on Windows 10 with all the latest bits installed.
I'm not sure what system-wide Java version was installed when I last looked into this.
Tobi users are not complaining / reporting about Pipeline failures, so I guess this is an isolated / edge case."

They'll keep us posted of any new problem reports but for now, I think it looks like it's working.
